### PR TITLE
[backport] Fix missing documents

### PR DESCRIPTION
**Pull request by okuta**
Backport of #1148

### CODE REVIEW

**Comment by kmaehashi** [comment]
Jenkins, test this please.

**Comment by kmaehashi** [comment]
Jenkins, test this please.

File: docs/source/reference/statistics.rst [hunk 1]
```diff
@@ -33,4 +34,5 @@ Histograms
    :toctree: generated/
    :nosignatures:
 
+   cupy.histogram
```
Comment (kmaehashi) [inline docs/source/reference/statistics.rst]: histogram (https://github.com/cupy/cupy/pull/1049) is not backported to v4.

**Comment by okuta** [comment]
Oh, sorry.

**Pull request closed without merging**